hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/processors/CustomMessageCallProcessor.java

```diff
@@ -164,6 +184,37 @@ public void start(@NonNull final MessageFrame frame, @NonNull final OperationTra
         frame.setState(MessageFrame.State.CODE_EXECUTING);
     }
 
+    /**
+     * Checks if the given message frame is a create scenario.
+     *
+     * <p>This method inspects the first four bytes of the input data of the message frame
+     * to determine if it matches any of the known selectors for creating fungible or non-fungible tokens.
+     *
+     * @param frame the message frame to check
+     * @return true if the input data matches any of the known create selectors, false otherwise
+     */
+    private boolean checkIfCreateScenario(MessageFrame frame) {
+        if (frame.getInputData().isEmpty()) {
+            return false;
+        }
+        var selector = frame.getInputData().slice(0, 4).toArray();
+        return Arrays.stream(new byte[][] {
+                    CREATE_FUNGIBLE_TOKEN_V1.selector(),
```

Comment:
Is there a way to have a single list of selectors to check here and in `CreateTranslator` so that we do not have to remember to maintain both lists if we add supported selectors?